module-services/service-cellular/ServiceCellular.cpp

```diff
@@ -923,10 +921,11 @@ auto ServiceCellular::receiveSMS(std::string messageNumber) -> bool
     }
 
     auto ucscSetRetries = 0;
-    while (ucscSetRetries < ucscSetMaxRetries) {
+    while (ucscSetRetries < at::AtCmdMaxRetries) {
         if (!channel->cmd(at::AT::SMS_UCSC2)) {
             ++ucscSetRetries;
             LOG_ERROR("Could not set UCS2 charset mode for TE. Retry %d", ucscSetRetries);
+            vTaskDelay(at::AtCmdRetryDelayMS);
```

Comment:
@niepiekm Please at least add a comment on why this delay is present here and what problem it solves.

Reply:
@mjan-dev This particular instance of delay is not actually needed, as the command hasn't caused problems so far. I just added it for consistency but will remove it.